### PR TITLE
Feature/banner ads

### PR DESCRIPTION
**Pull request by PeterZlodej**
[ch218]

### CODE REVIEW

File: common/v2/features/Dashboard/BannerAd/BannerAd.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+export default function BannerAd() {
+  const randomIndex = Math.floor(Math.random() * ads.length);
+  const ad = ads[randomIndex];
+  const externalLink = ad.url.startsWith('http');
+
+  return (
+    <AdWrapper onClick={() => onAdClick(ad.name)}>
+      {externalLink ? (
+        <a href={ad.url} target="_blank" rel="noreferrer">
```
Comment (vilsbole) [inline common/v2/features/Dashboard/BannerAd/BannerAd.tsx]: Nice! I didn't know that `noreferrer` also covers `noopener`

File: common/v2/features/Dashboard/BannerAd/BannerAd.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+};
+
+export default function BannerAd() {
+  const randomIndex = Math.floor(Math.random() * ads.length);
+  const ad = ads[randomIndex];
+  const externalLink = ad.url.startsWith('http');
+
+  return (
+    <AdWrapper onClick={() => onAdClick(ad.name)}>
+      {externalLink ? (
```
[the remark this answers is not in the document]
Comment (PeterZlodej) [reply common/v2/features/Dashboard/BannerAd/BannerAd.tsx]: Sure, fixed!

**Comment by PeterZlodej** [comment]
> I would like to reduce directory nesting and directory dependencies within the project.
> What do you think of:
> 
> 1. Since `BannerAds` can be reused in other places of the app we can move it to `v2/components`.
> 2. Since the images will only be used by `BannerAds` we can include them in the component directory

1. If we will reuse the ads component, that makes sense, therefore I moved the BannerAd component to v2/components. 

2. I believe that in this project assets are always stored in the /common/assets directory, in part because the current webpack config does not allow it. For consistency, we put the assets for banner ads in that directory. Maybe we can agree on a common approach where non-reusable assets are added to the component folders and apply that for all existing components across the project as well.

**Comment by vilsbole** [comment]
> Maybe we can agree on a common approach where non-reusable assets are added to the component folders and apply that for all existing components across the project as well.

Yes that makes sense. I'll ask to update the webpack config to be able to handle assets in component folders as well.

**Pull request merged**